src/main/java/school/hei/haapi/authentication/JwtConfiguration.java

```diff
@@ -0,0 +1,56 @@
+package school.hei.haapi.authentication;
+
+import java.io.Serializable;
+import org.springframework.stereotype.Component;
+
+@Component
+public class JwtConfiguration implements Serializable {
+  private static final String USER_POOL_ID = ("us-east-1_AvlIh6liI");
+  private static final String REGION = ("us-east-1");
+  private static final String HTTP_HEADER = "Authorization";
+  private static final int CONNECTION_TIMEOUT = 2000;
+  private static final int READ_TIMEOUT = 2000;
+  private String identityPoolId;
+  private String jwkUrl;
+  private String userNameField = "cognito:username";
+
+  public String getJwkUrl() {
+    return this.jwkUrl != null && !this.jwkUrl.isEmpty()
+        ? this.jwkUrl
+        : String.format(
+            "https://cognito-idp.%s.amazonaws.com/%s/.well-known/jwks.json",
+            JwtConfiguration.REGION, JwtConfiguration.USER_POOL_ID);
+  }
+
+  public String getCognitoIdentityPoolUrl() {
+    return String.format("https://cognito-idp.%s.amazonaws.com/%s", JwtConfiguration.REGION, JwtConfiguration.USER_POOL_ID);
```

Comment:
Use constant for the String. Eg: COGNITO_USER_POOL_URL_FORMAT. And refactor with above url. Eg: COGNITO_JWKS_URL_FORMAT = COGNITO_USER_POOL_URL_FORMAT + "..."